amp/parse.go

```diff
@@ -32,6 +36,14 @@ type Size struct {
 	Width          int64
 }
 
+// Policy consent types
+const (
+	NO_CONSENT = 0
+	TCF1       = 1
+	TCF2       = 2
+	CCPA       = 3
```

Comment:
These constants should be prefaces with something which ties them to the consent type and Go does not use SCREAMING_CAPS for constant names. Perhaps something like: ConsentNon/ConsentUndefined, ConsentTCF1, ConsentTCF2, ConsentUSPrivacy